File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/GameUpdatedMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.game.GameDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class GameUpdatedMessage extends BaseMessage {
    private GameDto game;

    public GameUpdatedMessage() {
        super(MessageType.GAME_UPDATED);
    }
}

File: front/src/app/core/domain/message/game-updated.message.ts
import {GameDto} from "@domain/games/game.dto";
import {BaseMessage} from "@domain/message/base.message";
import {MessageTypeEnum} from "@domain/message/message-type.enum";

export class GameUpdatedMessage extends BaseMessage {
  override readonly type = MessageTypeEnum.GAME_UPDATED;
  game: GameDto;
}
